lib/pronto/flay.rb

```diff
@@ -3,19 +3,27 @@
 
 module Pronto
   class Flay < Runner
+    # Pronto::Flay#run sets flay
+    attr_writer :flay
+
     def run
       files = ruby_patches.map(&:new_file_full_path)
-      files = ::Flay.filter_files(files)
 
       if files.any?
-        flay.process(*files)
+        # The current Flay (2.8.0) takes care of filtering
+        # files by looking at .flayignore.
+        # Saving the returned Flay object at @flay so we
+        # can inspect it and build the messages Array.
+        self.flay = ::Flay.run(files)
```

Comment:
Not a fan of this pattern with `attr_writer`. Maybe we can improve it to something like:

```ruby
def flay
  @flay ||= ::Flay.run(files)
end

def files
  ruby_patches.map(&:new_file_full_path)
end
```

This is now possible, because `ruby_patches` is just a `Runner` method 😄